ui/components/component-library/textarea/README.mdx

```diff
@@ -0,0 +1,148 @@
+import { Story, Canvas, ArgTypes } from '@storybook/addon-docs';
+
+import { Textarea } from './textarea';
+
+# Textarea
```

Comment:
Named `Textarea` instead of `TextArea` to follow the examples found in the audit Chakra and Material UI
![Screenshot 2023-12-13 at 5 52 15 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/b2f429b9-e9e0-423a-abdd-3c03d0edf927)